build/common/installer/scripts/tomlparser.rb

```diff
@@ -158,11 +158,6 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         @logEnableMultiline = parsedConfig[:log_collection_settings][:enable_multiline_logs][:enabled]
         puts "config::Using config map setting for multiline logging"
 
-        if @containerLogSchemaVersion.strip.casecmp("v2") != 0
-          puts "config:: WARN: container logs V2 is disabled and is required for multiline logging. Disabling multiline logging"
-          @logEnableMultiline = "false"
```

Comment:
Don't we still want to do this if someone has multline enabled but they changed configmap to use v1?

Reply:
any new onboarding is v2 and therefore checking is NOOP. however for existing v1 customers yes they can in a way enable multiline with this change however that is unsupported and not publicized. 

we need to remove this check because someone who onboards via UX now has v2 enabled via DCR and assuming they then enable multiline thinking v2 is already enabled it wont work